kpops/cli/pipeline_config.py

```diff
@@ -104,11 +105,16 @@ def customise_sources(
             init_settings: SettingsSourceCallable,
             env_settings: SettingsSourceCallable,
             file_secret_settings: SettingsSourceCallable,
-        ):
+        ) -> tuple[
+            SettingsSourceCallable,
+            SettingsSourceCallable,
+            Callable[["PipelineConfig"], dict | list],
```

Comment:
This is a crazy return type 😮 
```python
Callable[["PipelineConfig"], dict | list],
```
@disrupted is this really the return type?

Reply:
see https://github.com/bakdata/kpops/pull/353#discussion_r1316071748

Reply:
I think my main question is if we need to pas PipelineConfig as a string? 🧐

Reply:
https://github.com/bakdata/kpops/pull/353#discussion_r1316106194

Reply:
@sujuka99 can you change the string to the actual object?